cfme/containers/provider/__init__.py

```diff
@@ -85,6 +85,11 @@
     }
 )
 
+cfg_btn_add = {
```

Comment:
it seems to me this construction is used only in one place. So, it might be better off putting it to where place where it is used.